test/functorch/test_aotdispatch.py

```diff
@@ -6022,6 +6022,108 @@ def fn(x):
         out = torch.compile(fn, backend="aot_eager", fullgraph=True)(inp)
         self.assertEqual(ref_out, out)
 
+    def test_channels_last_grads_no_force_contiguous_dense(self):
```

Comment:
I would add some more tests with some outputs that are channels last, and some outputs that aren't.